test/connector-functions.test.js

```diff
@@ -0,0 +1,31 @@
+// Copyright IBM Corp. 2013,2016. All Rights Reserved.
+// Node module: loopback-connector-mongodb
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+// This test written in mocha+should.js
+var should = require('./init.js');
```

Comment:
Please use `const`, see http://loopback.io/doc/en/contrib/style-guide.html#variable-declarations

(According to package.json, we support only Node.js 4.x and newer, therefore `const` is allowed.)

Reply:
I addressed the feedback but causes eslint failure, its version is too old in this repo and a upgrading causes 100+failures and could not be fixed by running a `--fix` command, so I reverted the change and opened an issue for eslint upgrade:
https://github.com/strongloop/loopback-connector-mongodb/issues/408
I am extremely willing to dig more but don't get too much time for it due to the interrupt issue and my lb-next process is too behind, will get back to it when everything's on the right track. Sorry for that.